tests/resources/test_analysis_workflow.py

```diff
@@ -0,0 +1,183 @@
+import random
+import uuid
+
+import pytest
+
+from citrine.informatics.workflows.analysis_workflow import AnalysisWorkflow
+from citrine.resources.analysis_workflow import AnalysisWorkflowCollection
+
+from tests.utils.factories import AnalysisWorkflowEntityDataFactory
+from tests.utils.session import FakeCall, FakeSession
+
+
+def paging_response(*items):
+    return {"response": items}
+
+
+def _assert_user_timestamp_equals_dict(user, time, ut_dict):
+    assert str(user) == ut_dict['user']
+    assert int(time.timestamp()) == int(ut_dict['time'] / 1000)
+
+
+def _assert_aw_plot_equals_dict(plot, plot_dict):
+    # Presently, plots are just raw JSON, so we can compare them directly.
+    assert plot == plot_dict
```

Comment:
Not just having this be an in-line assert because we expect this to normally be checking multiple things?

Reply:
I initially just set it up this way as part of a pattern before realizing there was nothing I needed to do for now. It could be useful if we ever add plot operations to the SDK. But for now, it doesn't feel important to collapse it.